frontend/src/pages/SettingsRouter/SettingsRouter.tsx

```diff
@@ -31,7 +33,19 @@ export type WorkspaceSettingsTab =
 	| 'settings'
 	| 'current-plan'
 	| 'upgrade-plan'
-	| SettingGroups
+
+export const isSettingsPath = (path: string[]) => {
```

Comment:
This always takes in a `path` value from `useLocation`, so it can easily be refactored into a custom hook to better encapsulate all of it's dependent logic.